markdownlint/docs/TOP009.md

```diff
@@ -0,0 +1,25 @@
+# `TOP009` - Lesson overview items sentence structure
+
+Tags: `content`, `language`
```

Comment:
```suggestion
Tags: `content`
```
Since `content` is the only tag in the rule (and `language` is more for the languages used like code block language)